test/Sema/accessibility_package_contextual_keyword.swift

```diff
@@ -0,0 +1,64 @@
+// RUN: %empty-directory(%t)
+// RUN: %{python} %utils/split_file.py -o %t %s
+
+// RUN: %target-swift-frontend-typecheck -verify -disable-availability-checking %t/main.swift -package-name myPkg
+// RUN: %target-swift-frontend-typecheck -verify -disable-availability-checking %t/A.swift -package-name myPkg
+// RUN: %target-swift-frontend-typecheck -verify -disable-availability-checking %t/B.swift -package-name myPkg
+// RUN: %target-swift-frontend-typecheck -verify -disable-availability-checking %t/D.swift -package-name myPkg
+// RUN: not %target-swift-frontend-typecheck -verify -disable-availability-checking %t/C.swift -package-name myPkg 2>&1 | %FileCheck %s
+
+// BEGIN main.swift
+package(set) public var a: String // should pass when `package` modifier is used at top level decls
+public package(set) var b: String
+package let c: Int
+package var d: Int
+package func f() {}
```

Comment:
Would calling a function named `package` still be accepted at the top level?

```
func package() {}
package()
```

Reply:
yes it works. I'll add them to the test